pkgs/desktops/mate/mate-session-manager/default.nix

```diff
@@ -5,11 +5,11 @@
 
 stdenv.mkDerivation rec {
   pname = "mate-session-manager";
-  version = "1.22.2";
+  version = "1.22.3";
 
   src = fetchurl {
     url = "http://pub.mate-desktop.org/releases/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
```

Comment:
this redirected me to https://pub.mate-desktop.org/. can you use https?

Reply:
Done for all mate packages.